docs/CHANGELOG.rst

```diff
@@ -19,6 +19,10 @@ Fixed
   (referended in ``src`` attribute) with no output field
 * Make ``Data.annotation`` an instance attribute instead of class
   attribute
+* Remove annotation tests from ``test_macs``
+* Fix ``get_*`` calls in tests by including species and build inputs
+* Remove/replace invalid collection references in ``get_*`` calls
+* Update cuffquant ``gff`` input to ``annotation`` in helper and test functions
```

Comment:
This does not belong into "fix e2e tests" commit. Please move it to the separate commit and mark this change as backward incompatible.